linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/FunctionEmitter.scala

```diff
@@ -450,7 +450,7 @@ private[emitter] class FunctionEmitter(sjsGen: SJSGen) {
         implicit pos: Position): js.Ident = {
 
       val recIdent = (tree.record: @unchecked) match {
-        case VarRef(ident)                 => transformLocalVarIdent(ident)
+        case VarRef(name)                  => transformLocalVarIdent(name)
```

Comment:
I don't think this is correct. `transformLocalVarIdent` now takes the record select position, not the position of the var ref.

To prevent this, consider `transformVarRef`? (IDK if this matters in practice given how the optimizer emits recordselects, but it still seems wrong :shrug: )

Reply:
Ah, good catch! I turned it into a `transformLocalVarRefIdent(varRef: VarRef): js.Ident` to avoid that issue. There is only one call site where we actually wrap it in a `js.VarRef`, so a `transformLocalVarRef(VarRef): js.VarRef` would not have helped per se.